ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js

```diff
@@ -0,0 +1,268 @@
+/*!
+ * ReadRemaining.js 1.0.0
```

Comment:
this line said the version of this lib is 1.0.0, I think you need to check this with the author.